intrastat_product/models/account_move.py

```diff
@@ -77,12 +82,16 @@ def compute_intrastat_lines(self):
 
     def _get_intrastat_line_vals(self, line):
         vals = {}
+        notedict = {
+            "note": "",
+            "line_nbr": 0,
+        }
         decl_model = self.env["intrastat.product.declaration"]
         if decl_model._is_product(line):
             hs_code = line.product_id.get_hs_code_recursively()
             if not hs_code:
                 return vals
-            weight, qty = decl_model._get_weight_and_supplunits(line, hs_code)
+            weight, qty = decl_model._get_weight_and_supplunits(line, hs_code, notedict)
```

Comment:
IMO isn't necessary because in `intrastat_product` addon only have 2 params.
https://github.com/OCA/intrastat-extrastat/blob/13.0/intrastat_product/models/intrastat_product_declaration.py#L268

Reply:
Well, that is the 13.0 version :sweat_smile: 
This is the current method: https://github.com/OCA/intrastat-extrastat/blob/14.0/intrastat_product/models/intrastat_product_declaration.py#L327